scripts/python/PyAutoscoper/connect.py

```diff
@@ -130,24 +179,25 @@ def loadTrackingData(
         :type is_rad: bool
         :param interpolate: Optional - If true, the tracking data will be interpolated using the spline method. If false, the tracking data will be saved as is (with NaN values). Defaults to false.
         :type interpolate: bool
-        :raises Exception: If the tracking data file is not found, or If the server fails to load the tracking data
+        :raises AutoScoperServerError: If the server fails to load the tracking data
+        :raises AutoscoperConnectionError: If the connection to the server is lost
         """
         if self.verbose:
             print(f"Loading tracking data: {tracking_data}")
-        if not os.path.exists(tracking_data):
```

Comment:
I suggest to keep the existing logic flow testing for pre-conditions and exiting/raising if not valid.

In this case, `Exception` should be switched with `AutoScoperServerError` and the class to `self.closeConnection()` removed.

The user of PyAutoscoper may decide to ignore the exception and report a warning, we should not do so here.